internal/signer/darwin/keychain/keychain_test.go

```diff
@@ -46,3 +50,147 @@ func TestBytesToCFDataRoundTrip(t *testing.T) {
 		t.Errorf("bytesToCFData -> cfDataToBytes\ngot  %x\nwant %x", got, want)
 	}
 }
+
+func TestEncryptRSA(t *testing.T) {
+	hashFunc := sha256.New()
+	rng := rand.Reader
+	key, errCred := Cred("enterprise_v1_corp_client-signer-0-2018-07-03T10:55:10-07:00 K:1, 2:BXmhnePmGN4:0:18")
+	if errCred != nil {
+		t.Errorf("Cred error: %q", errCred)
+		return
+	}
+	message := []byte("Plain text to encrypt")
+
+	_, encryptErr := key.EncryptRSA(hashFunc, rng, message)
+	if encryptErr != nil {
+		t.Errorf("Encrypt error: %q", encryptErr)
+		return
+	}
+	fmt.Println("Encrypted")
+}
+
+func BenchmarkEncryptRSA(b *testing.B) {
+	hashFunc := sha256.New()
+	rng := rand.Reader
+	key, errCred := Cred("enterprise_v1_corp_client-signer-0-2018-07-03T10:55:10-07:00 K:1, 2:BXmhnePmGN4:0:18")
+	if errCred != nil {
+		b.Errorf("Cred error: %q", errCred)
+		return
+	}
+	message := []byte("Plain text to encrypt")
+
+	for i := 0; i < b.N; i++ {
+		_, encryptErr := key.EncryptRSA(hashFunc, rng, message)
+		if encryptErr != nil {
+			b.Errorf("Encrypt error: %q", encryptErr)
+			return
+		}
+	}
+}
+
+func TestEncrypt(t *testing.T) {
+	key, err := Cred("enterprise_v1_corp_client-signer-0-2018-07-03T10:55:10-07:00 K:1, 2:BXmhnePmGN4:0:18")
+	if err != nil {
+		t.Errorf("Cred error: %q", err)
+		return
+	}
+	plaintext := []byte("Plain text to encrypt")
+	ciphertext, encryptErr := key.Encrypt(plaintext)
+	if encryptErr != nil {
+		t.Errorf("Encrypt error: %v", encryptErr)
+		return
+	}
+	fmt.Printf("Encrypted %+v\n", ciphertext)
+}
+
+func BenchmarkEncrypt(b *testing.B) {
+	key, err := Cred("enterprise_v1_corp_client-signer-0-2018-07-03T10:55:10-07:00 K:1, 2:BXmhnePmGN4:0:18")
+	if err != nil {
+		b.Errorf("Cred error: %q", err)
+		return
+	}
+	plaintext := []byte("Plain text to encrypt")
+	for i := 0; i < b.N; i++ {
+		key.Encrypt(plaintext)
+	}
+}
+
+func TestDecrypt(t *testing.T) {
+	key, err := Cred("enterprise_v1_corp_client-signer-0-2018-07-03T10:55:10-07:00 K:1, 2:BXmhnePmGN4:0:18")
+	if err != nil {
+		t.Errorf("Cred error: %q", err)
+		return
+	}
+	byteSlice := []byte("Plain text to encrypt")
+	ciphertext, _ := key.Encrypt(byteSlice)
+	plaintext, decryptErr := key.Decrypt(ciphertext)
+	if decryptErr != nil {
+		t.Errorf("Decrypt error: %v", decryptErr)
+		return
+	}
+	plainString := string(plaintext)
+	fmt.Printf("Decrypted %+v\n", plainString)
```

Comment:
Maybe we could compare `plainString` to `byteSlice` and assert they are the same?

Then remove the print statement.

Reply:
A note: I compared plaintext ([]byte) to byteSlice ([]byte) instead of plainString (string) due to type conflicts

Reply:
Ah, thanks for catching that!